src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -119,6 +120,14 @@
     boolean isChatColorsEnabled();
 
     /**
+     * Simulates a chat message from a player.
+     *
+     * @param message The message to send
+     * @return The event thrown from the message
+     */
+    MessageChannelEvent.Chat chat(Text message);
```

Comment:
That would however limit what the plugin can do. It feels like a severe limitation for no reason other than to prevent them doing something the vanilla client can't.